windows/src/engine/kmcomapi/kmcomapi.dpr

```diff
@@ -81,10 +81,6 @@ uses
   custinterfaces in '..\..\global\delphi\cust\custinterfaces.pas',
   utilkeymancontrol in 'util\utilkeymancontrol.pas',
   klog in '..\..\global\delphi\general\klog.pas',
-  DCPcrypt2 in '..\..\ext\dcpcrypt\DCPcrypt2.pas',
-  DCPbase64 in '..\..\ext\dcpcrypt\DCPbase64.pas',
-  DCPconst in '..\..\ext\dcpcrypt\DCPconst.pas',
-  DCPrc4 in '..\..\ext\dcpcrypt\DCPrc4.pas',
```

Comment:
`DCPrc4` is the one we're keeping with #6694, right?  Granted, the build passed... but are we sure it's safe to eliminate from this component?

Reply:
If the build passed, then the components are unused :grin:. kmcompapi does not use the only unit which uses DCPrc4.pas: GlobalProxySettings.pas, as it has no Internet connectivity, so it doesn't need these units.

Reply:
> If the build passed, then the components are unused 😁.

While often true, I'm just going to point out that two of my earlier comments (especially the first) on this PR point to ways where this could have been untrue if the IDE did things just a bit differently.  (Debug configs are important, too.)

I think they also point out a few reasons why it's not totally self-evident.  I'm not super-well-versed in the .dproj file format, after all.

Reply:
Yeah, never said it was self-evident. Not expecting you to be an expert in .dpr files (.dpr files are the source of truth for which files are in a Delphi project, not .dproj, thanks very much Delphi for two files with overlapping content). But you can trust me: if it's referenced in the .dpr, and the file is missing, the build will fail.